docs/api/IoT.js-API-Net.md

```diff
@@ -108,61 +273,192 @@ This class is used to create a TCP or local server. You can create `net.Server`
 ## Prototype Funtions
```

Comment:
@robertsipka Could you please fix the typos (Funtions -> Functions)  although it is not introduced in this PR?